formula/src/main/java/com/instacart/formula/RuntimeExtensions.kt

```diff
@@ -1,17 +1,22 @@
+@file:Suppress("DeprecatedCallableAddReplaceWith")
```

Comment:
Supressing `ReplaceWith`, since the extension method is becoming a method on the class, there is no replacement to make, just a deletion of an import (which unfortunately, `ReplaceWith` does not cover).